index.js

```diff
@@ -33,6 +33,18 @@ function normalize(type, value) {
   return result;
 }
 
+function conforms(type, value) {
```

Comment:
I think multiple types is common enough and more clearly conveyed by an array than by a function (`function(value) { return typeof value === "string" || typeof ...`). I also like supporting the string, because if I have the type in a variable, I would prefer `normalize(type, ...)` to `normalize[type](...)`.

I don't go for flexibility at all cost however I think the overhead of supporting the 3 case is quite light here.